client/release/scripts/release.sh

```diff
@@ -22,6 +22,27 @@ cleanup() {
     exit 1   
 }
 
+# waits for NPM to publish on internal registry to update package json
+wait_for_npm_publish(){
+    package_name="${1}"
+
+    # Assign the correct version based off if its core or not
+    updated_npm_version=$( [[ "$package_name" == *"core"* ]] && echo "$version_sdk_core" || echo "$version_sdk" )
+    
+    while true; do
+    npm_verison=$(npm view "$package_name" version)
+    # if package matches npm registry than break out of loop and update package.json
+    if [ "$npm_verison" == "$updated_npm_version" ]; then
+        echo "The package version $npm_verison is up to date!"
+        break
+    else
+        # Sleep for another 5 seconds if it still didnt sync
+        echo "The package version $npm_verison is not up to date. Latest version is $updated_npm_version."
```

Comment:
Let's do something more along the lines of

```suggestion
        echo "The $updated_npm_version package version is not yet published to NPM. Waiting..."
```

to instruct the developer that they have nothing to do, but it's just a matter of waiting.